lib/components/Dialog/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from "react"
+import { Modal } from "../Modal"
+import { DialogHeader } from "./DialogHeader"
+import { DialogBody } from "./DialogBody"
+import { DialogFooter } from "./DIalogFooter"
+import "./Dialog.scss"
+
+type TDialogProps = {
+  bodyChildren:ReactNode
+  footerChildren:ReactNode;
+  title?: ReactNode;
+}
+
+export const Dialog = ({ bodyChildren,footerChildren,title }: TDialogProps) => {
+  return (
+    <Modal className={"deriv-dialog"} isOpen={true} ariaHideApp={false} >
+      <DialogHeader title={title} />
+      <DialogBody children={bodyChildren}/>
+      <DialogFooter children={footerChildren}/>
+    </Modal>
+  )
+}
```

Comment:
```suggestion
export const Dialog = ({ bodyChildren,footerChildren,title }: TDialogProps) => {
  return (
    <Modal className={"deriv-dialog"} isOpen={true} ariaHideApp={false} >
         {children}
    </Modal>
  )
}

Dialog.Header = DialogHeader;
Dialog.Body = DialogBody;
Dialog.Footer = DialogFooter;
Dialog.setAppElement = Modal.setAppElement;
```
can we keep it the same as Modal?